src/chapters/thymeleaf-views/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _thymeleaf:
+
+Thymeleaf Views
+================
+
+.. toctree::
+   :maxdepth: 3
+
+   why-templates
+   create-template
+   using-templates
+   template-iteration
+   template-conditionals
```

Comment:
I think these two should be switched to match the order in which we teach other new languages. Also, ``th:block`` is newer concept than ``th:if`` and ``th:unless``.

Reply:
Done.  The order in the TOC follows the one shown in the LC101 objectives.  Should we rearrange those as well?

Reply:
Got it.  Thanks.